.github/workflows/fedora-copr-build.yml

```diff
@@ -44,40 +46,19 @@ jobs:
           echo "project_yesterday=$username/llvm-snapshots-incubator-$yesterday" >> $GITHUB_ENV
           echo "project_target=$username/llvm-snapshots" >> $GITHUB_ENV
 
-          cat <<EOF > ~/functions.sh
-          set -x
-          # set -e
-          # TODO(kwk): Is there a better way to check project existence?
-          # TODO(kwk): Maybe: copr list $username | grep --regexp="^Name: \$project$"
-          function project_exists(){
-            local project=\$1;
-            copr get-chroot \$project/fedora-rawhide-x86_64 > /dev/null 2>&1 \
-            && echo "true" || echo "false";
-          }
-
-          function get_active_build_ids(){
-            local project=\$1;
-            copr list-builds --output-format text-row \$project \
-              | grep --perl-regexp --regexp='(running|waiting|pending|importing|starting)' \
-              | cut -f 1
-          }
-          EOF
-
       - uses: actions/checkout@v3
 
       - name: "Check for Copr projects existence (yesterday, today, target)"
         shell: bash -e {0}
         run: |
-          source ~/functions.sh
+          source github/functions.sh
 
-          # Check if yesterday's project exists and is waved through
+          # Check if yesterday's project exists and all builds succeeded
           yesterdays_project_exists=`project_exists ${{ env.project_yesterday }}`
           if [[ "$yesterdays_project_exists" == "true" ]]; then
-            url=https://copr.fedorainfracloud.org/coprs/`echo ${{ env.project_yesterday }} | sed -s 's/^@/\/g\//'`
-            curl -sL -o page.html $url 2>&1
-            grep "`head -n1 project-description.md`" page.html \
-              && yesterdays_project_exists="false" \
-              || true
+            if has_not_succeeded_builds ${{env.project_yesterday}}; then
```

Comment:
You've renamed the function, but the old name is still used here.

Reply:
Thank you. Fixed in bee20b0.